backend/src/test/java/com/github/redstylzz/backend/service/CategoryServiceTest.java

```diff
@@ -78,20 +81,44 @@ void shouldReturnListOnSuccessfulRename() {
         when(repository.findByUserIDAndCategoryID(anyString(), anyString())).thenReturn(TestDataProvider.testCategory());
         when(repository.existsByUserIDAndCategoryName(anyString(), anyString())).thenReturn(false);
 
-        underTest.renameCategory(user, "", "");
+        List<Category> wantedList = underTest.renameCategory(user, "", "");
```

Comment:
a more common naming in tests is actual vs expected - wanted list sounds like a poster a wild west movie